lang/cs/Org.Apache.REEF.Wake/Remote/Impl/RemoteConnectionRetryHandler.cs

```diff
@@ -16,7 +16,7 @@
 // under the License.
 
 using System;
-using Microsoft.Practices.TransientFaultHandling;
```

Comment:
I realize that this will probably break the old build. I'm working on a solution.